interface/wx/filedlg.h

```diff
@@ -264,6 +264,9 @@ class wxFileDialog : public wxDialog
 
     /**
         Returns the default filename.
+
+        @note This function should not be used with dialogs which have the @c wxFD_MULTIPLE style,
```

Comment:
```suggestion
        @note This function can't be used with dialogs which have the @c wxFD_MULTIPLE style,
```
Getting an assert is stronger than "shouldn't".

Reply:
Fixed in new version.